### ISSUE TITLE
[symfony issue] composer update to regenerate fresh vendor directory produces locale_en issue (on quick export and export via connector)

### ISSUE THREAD

**Issue by wakqasahmed** [akeneo/pim-community-dev#5300]
**I'm reporting a Bug**

This was a bug in symfony and it has been resolved for 2.7 version and onwards.

https://github.com/symfony/symfony/pull/19751

Release 2.7.17
https://github.com/symfony/symfony/pull/19820

Here is the error:

```
CSV PRODUCT QUICK EXPORT  The Symfony\Component\Intl\NumberFormatter\NumberFormatter::__construct() method's argument $locale value 'en_US' behavior is not implemented. Only the locale "en" is supported. Please install the "intl" extension for full localization capabilities.
```

**Comment by fabienlem** [comment]
Hello @wakqasahmed,

We know that some of our vendors should be updated. In order to achieve this, we have to keep in mind the following matters:

- If we upgrade our vendors in a patch version, it can introduce some backwards compatibility breaks with strong impacts on the current PIM projects. It has already happened in the past, so we decided to upgrade our vendors only for minor releases 1.**X** (not for patch versions 1.6.**X**).
- One major issue for PIM users is the performance. Upgrading vendors can improve them or... damage them. We have to process many tests, especially to make sure all batch operations are as fast as before.
- Last thing, we have to rework all our automated tests. As an example, upgrading Symfony to 2.7.17 version causes 119 scenarios to fail (we have approximately 1700 scenarios to maintain).

These tasks can be very time consuming, and we are aware that it's necessary. For now, you can suggest a pull request with an upgraded "composer.json" file on the **master** branch. We will later perform tests and adjustments I mentioned, but I can't tell you in which upcoming PIM version it will be available.

Regards,
Fabien

**Comment by willy-ahva** [comment]
I close. Fixed with #5521. Thanks @wakqasahmed !
